storages/redis-storage/src/lib.rs

```diff
@@ -0,0 +1,459 @@
+#![deny(clippy::str_to_string)]
+
+mod alter_table;
+mod index;
+mod metadata;
+mod transaction;
+
+use {
+    async_trait::async_trait,
+    futures::stream::iter,
+    gluesql_core::{
+        chrono::Utc,
+        data::{CustomFunction as StructCustomFunction, Key, Schema, Value},
+        error::{Error, Result},
+        store::{CustomFunction, CustomFunctionMut, DataRow, RowIter, Store, StoreMut},
+    },
+    redis::{Commands, Connection},
+    std::cell::RefCell,
+    std::collections::BTreeMap,
+};
+
+pub struct Item {
+    pub schema: Schema,
+}
```

Comment:
This code block looks currently not being used. it would be good to remove this

Reply:
Done